ztp/resource-generator/entrypoints/generator

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bash
+
+PROG=$(basename $0)
+SUB_COMMAND=$1
+
+# sub command-line arguments
+EMS_ONLY="false"
+WRAP_IN_POLICY="true"
+
+ztpTmpDir="/home/ztp/tmp"
+basePath="/kustomize/plugin/ran.openshift.io/v1"
+policyGenPath=${basePath}/policygentemplate
+siteConfigPath=${basePath}/siteconfig
+
+printenv MOUNT_DIR &> /dev/null
+if [[ $? -ne 0 ]]; then
```

Comment:
Another way to do this is to use variable substitution:
`MOUNT_DIR=${MOUNT_DIR:-/home/ztp_resources}`